src/main/java/Duke/Duke.java

```diff
@@ -0,0 +1,84 @@
+package Duke;
+
+import Duke.Command.*;
+import Duke.Parser.Parser;
+import Duke.Storage.Storage;
+import Duke.Task.*;
+import Duke.Exception.*;
+import Duke.TaskList.TaskList;
+import Duke.Ui.Ui;
+
+import java.io.*;
```

Comment:
Avoid wildcard imports. It might be better to explicitly import the packages you're using.